atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -398,8 +321,8 @@ const CreateCrashRecord = ({ client }) => {
                 id="primary-address-input"
                 name="primary-address-input"
                 placeholder="ex: S 900 AUSTIN AVE"
-                value={primaryAddress}
-                onChange={e => setPrimaryAddress(e.target.value)}
+                value={primaryStreetName?.toUpperCase()}
```

Comment:
i am getting this classic error when i type into this field: `A component is changing an uncontrolled input of type text to be controlled`.

i think the problem is that `{primaryStreetName?.toUpperCase()` returns `undfined`. if you change this to `{primaryStreetName?.toUpperCase() || ""`} it should fix the problem